src/cpp/fastdds/builtin/type_lookup_service/TypeLookupManager.hpp

```diff
@@ -24,13 +24,13 @@
 #include <vector>
 #include <mutex>
 
-#include <fastdds/builtin/typelookupservice/TypeLookupRequestListener.hpp>
-#include <fastdds/builtin/typelookupservice/TypeLookupReplyListener.hpp>
-#include <fastdds/builtin/typelookupservice/common/TypeLookupTypes.hpp>
-
 #include <fastdds/rtps/builtin/data/WriterProxyData.h>
 #include <fastdds/rtps/builtin/data/ReaderProxyData.h>
 
+#include <fastdds/builtin/type_lookup_service/common/TypeLookupTypes.hpp>
+#include <fastdds/builtin/type_lookup_service/TypeLookupRequestListener.hpp>
+#include <fastdds/builtin/type_lookup_service/TypeLookupReplyListener.hpp>
```

Comment:
It should be sorted alphabetically. Please, apply in the other files.

Reply:
Fixed in [7cbeaf0](https://github.com/eProsima/Fast-DDS/pull/4053/commits/7cbeaf000b9780d2985f5b663a0244701a6b5cbe)